### ISSUE TITLE
[BUG]: Copy Note

### ISSUE THREAD

**Issue by silidev** [ankidroid/Anki-Android#16964]
### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [X] This bug does not occur in the latest version of Anki

### What are the steps to reproduce this bug?

1. Open a card in study mode
2. Hit the edit button
3. Choose "Copy note"
4. Change something.
5. Save
You are back at the original note still in edit mode.
6. Save

### Expected behaviour

The card is saved.

### Actual behaviour

A window with "java.lang.NullPointerException" appears.

### Debug info

```text
AnkiDroid Version = 2.18.4 (f867da96b8304edaebd0361326316bc7514f8cde)

Backend Version = 0.1.38-anki24.04.1 (24.04.1 ccd9ca1a8309b80bcb50ddc5d99c7ce63440bce9)

Android Version = 14 (SDK 34)

ProductFlavor = play

Manufacturer = samsung

Model = SM-S926B

Hardware = s5e9945

Webview User Agent = Mozilla/5.0 (Linux; Android 14; SM-S926B Build/UP1A.231005.007; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/127.0.6533.103 Mobile Safari/537.36

ACRA UUID = e48147db-e59c-4853-929c-905d382dd7e3

FSRS Enabled = true

Crash Reports Enabled = true
```

### (Optional) Anything else you want to share?

This is a regression in 2.18.4 which was not present in 2.18.0

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by david-allison** [comment]
@silidev Could you try the 2.19 beta parallel release? We've fixed a number of bugs in this area

**Comment by silidev** [comment]
Thanks, I will try it.
--
Viele Grüße - Helge

David Allison ***@***.***> schrieb am Mo., 26. Aug. 2024,
22:20:

**Comment by silidev** [comment]
Hi David! The bug seems fixed in 2.9 beta. I cannot reproduce it. You can
close this. Thank you! 😀

David Allison ***@***.***> schrieb am Mo., 26. Aug. 2024,
22:20:

**Comment by david-allison** [comment]
Thanks! We've moved to beta, so 2.19 will be out soon
